pytest_fixtures/sys_fixtures.py

```diff
@@ -21,3 +23,12 @@ def relax_bfa(default_sat):
     """Relax BFA protection against failed login attempts"""
     if default_sat:
         default_sat.cli.Settings.set({'name': 'failed_login_attempts_limit', 'value': '0'})
+
+
+@pytest.fixture(autouse=True, scope='session')
+def proxy_port_range(default_sat):
+    """Assigns port range for fake_capsules"""
+    port_pool_range = settings.fake_capsules.port_range
+    port_range = port_pool_range.replace(', ', '-')
```

Comment:
Can anyone let me why we keep the port range like `9091, 14999` instead of `9091-14999`?

Reply:
@devendra104 
I believe there were historical reasons - mainly the way the original `robottelo.properties` ini config parser worked - a config value in CSV format was automatically parsed into a list by the INI reader, while the "-"-separated value was parsed as a string.
This is no longer used so we can use whatever format we find fit.

Reply:
@devendra104  @rplevka  So can we update the ports range format in "fake_capsules.yaml"?

Reply:
@shweta83 absolutely - please provide an update to the template here:
https://github.com/SatelliteQE/robottelo/blob/master/conf/fake_capsules.yaml.template#L6

And then make the update to our internal repo to match.

Reply:
Feel free to consider options to how you define this - if you want to separately define `start` and `end` values, or supply them as a string with a dash.

Reply:
Thanks, @rplevka for the detailed information.
@shweta83 I think the port range format would be much appropriate here but it's up to you, I am ok with both.